gdb-multi-arch/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: '4'
+  number: '5'
```

Comment:
Not sure if this is needed or not?

Reply:
We should do it, so that when we build and upload all the new versions, people who have installed this will be able to detect the new version.

Reply:
Oops, we should revert this to 0 since we updated the version. I'll update that.